### PR TITLE
Remove goto label 6DCE02

### PR DESCRIPTION
**Pull request by frutiemax**

### CODE REVIEW

File: src/openrct2/ride/Vehicle.h [hunk 1]
```diff
@@ -373,6 +373,8 @@ struct Vehicle : EntityBase
     void UpdateLandscapeDoor() const;
     void UpdateLandscapeDoorBackwards() const;
     int32_t CalculateRiderBraking() const;
+
+    void loc_6DCE02(Ride* curRide);
```
Comment (Gymnasiast) [inline src/openrct2/ride/Vehicle.h:377]:
```suggestion
    void Loc6DCE02(const Ride& curRide);
```

File: src/openrct2/ride/Vehicle.h [hunk 2]
```diff
@@ -373,6 +373,8 @@ struct Vehicle : EntityBase
     void UpdateLandscapeDoor() const;
     void UpdateLandscapeDoorBackwards() const;
     int32_t CalculateRiderBraking() const;
+
+    void Sub6DCE02(const Ride& curRide);
```
Comment (Gymnasiast) [inline src/openrct2/ride/Vehicle.h:377]: Please rename this `Loc6DCE02` as I requested, not `Sub6DCE02`. It wasn’t a subroutine, otherwise it wouldn’t have been a goto in the first place.

File: src/openrct2/ride/Vehicle.cpp [hunk 3]
```diff
@@ -8577,24 +8579,27 @@ void Vehicle::UpdateTrackMotionMiniGolfVehicle(Ride* curRide, rct_ride_entry* ri
...
-        if (trackType != TrackElemType::EndStation)
-        {
-            return;
-        }
+        return;
+    }
+    _vehicleMotionTrackFlags |= VEHICLE_UPDATE_MOTION_TRACK_FLAG_3;
+    if (trackType != TrackElemType::EndStation)
+    {
+        return;
```
Comment (tupaschoal) [inline src/openrct2/ride/Vehicle.cpp:8602]: Any reason for removing the `{}`? They were adding scope to `trackType` and `ted` which I understand might be an overkill, but seems unrelated to the `goto` change.

**Review by tupaschoal — COMMENTED** [review]
Any hint at what we might call this function instead? Sounds to be something related to vehicles at the station.

File: src/openrct2/ride/Vehicle.cpp (hunk 3 above)
Comment (Gymnasiast) [reply src/openrct2/ride/Vehicle.cpp:8602]: They probably used to be necessary when it was still a loc, but are not now.

**Review by Gymnasiast — APPROVED** [review]
